src/System.Console/tests/TermInfo.cs

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using Xunit;
+
+public class TermInfo
+{
+    [Fact]
+    [PlatformSpecific(PlatformID.AnyUnix)]
+    public void VerifyInstalledTermInfosParse()
+    {
+        foreach (string location in ConsolePal.TermInfo.Database._terminfoLocations)
+        {
+            if (!Directory.Exists(location))
+                continue;
+
+            foreach (string term in Directory.EnumerateFiles(location, "*", SearchOption.AllDirectories))
+            {
+                if (term.ToUpper().Contains("README")) continue;
+                ConsolePal.TerminalColorInfo info = new ConsolePal.TerminalColorInfo(ConsolePal.TermInfo.Database.ReadDatabase(Path.GetFileName(term)));
+
+                if (string.IsNullOrEmpty(info.ForegroundFormat) == false)
+                {
+                    Assert.NotEmpty(ConsolePal.TermInfo.ParameterizedStrings.Evaluate(info.ForegroundFormat, 0 /* irrelevant, just an integer to put into the formatting*/));
+                }
+
+                if (string.IsNullOrEmpty(info.BackgroundFormat) == false)
+                {
+                    Assert.NotEmpty(ConsolePal.TermInfo.ParameterizedStrings.Evaluate(info.ForegroundFormat, 0 /* irrelevant, just an integer to put into the formatting*/));
+                }
+            }
+        }
+    }
+
+    [Theory]
+    [PlatformSpecific(PlatformID.AnyUnix)]
+    [InlineData("xterm-256color", "\u001B\u005B\u00330m", "\u001B\u005B\u00340m")]
+    [InlineData("screen", "\u001B\u005B\u00330m", "\u001B\u005B\u00340m")]
+    [InlineData("Eterm", "\u001B\u005B\u00330m", "\u001B\u005B\u00340m")]
+    [InlineData("wsvt25", "\u001B\u005B\u00330m", "\u001B\u005B\u00340m")]
+    [InlineData("mach-color", "\u001B\u005B\u00330m", "\u001B\u005B\u00340m")]
+    public void TermInfoVerification(string termToTest, string expectedForeground, string expectedBackground)
+    {
+        ConsolePal.TerminalColorInfo info = new ConsolePal.TerminalColorInfo(ConsolePal.TermInfo.Database.ReadDatabase(termToTest));
+        Assert.Equal(expectedForeground, ConsolePal.TermInfo.ParameterizedStrings.Evaluate(info.ForegroundFormat, 0));
+        Assert.Equal(expectedBackground, ConsolePal.TermInfo.ParameterizedStrings.Evaluate(info.BackgroundFormat, 0));
```

Comment:
Consider using a value other than 0, something that's not likely to be confused with a default value.  Some of the evaluation strings also do special processing that distinguishes between the bold and not-bold color values, so you might consider using a value from each range.